### DISCUSSION TITLE
Jキーのキーソケットのパッドが剥がれてしまい、キーを押下しても反応しなくなってしまいました。解決策を教えていただきたいです。

### DISCUSSION THREAD

**Discussion by take20m** [Yowkees/keyball#248] [Q&A]
![image](https://github.com/Yowkees/keyball/assets/40426755/5dc163ec-158b-4d95-bdc6-823a8a17b390)

**Comment by Yowkees** [comment]
![KB44Jfix](https://github.com/Yowkees/keyball/assets/84209772/3135fc15-cadc-40e0-9ed1-430078942d7f)
ご連絡ありがとうございます。
添付画像の緑点どうしをエナメル線やゴム被覆導線等で接続していただければ復活できると思います。
お手数ですがご対応お願いいたします。

**Reply by take20m** [reply]
ありがとうございます！
できました！
